src/commands/Preferences.js

```diff
@@ -9,27 +9,32 @@ export default function (context) {
     const preferences = getUserPreferences()
     var accessory = NSView.alloc().initWithFrame(NSMakeRect(0, 0, 300, 275))
```

Comment:
oops, I completely missed that while visually checking if it worked. Fixed 😄  .

Also, I was not sure about the phrasing for the checkbox label but hopefully what is in there is concise and still descriptive.